internal/server/storage/drivers/utils.go

```diff
@@ -966,12 +966,9 @@ func clearDiskData(diskPath string, disk *os.File) error {
 	}
 
 	if linux.IsBlockdev(st.Mode()) {
-		// If dealing with a block device, discard its current content.
+		// If dealing with a block device, attempt to discard its current content.
 		// This saves space and avoids issues with leaving zero blocks to their original value.
-		_, err = subprocess.RunCommand("blkdiscard", diskPath)
-		if err != nil {
-			return err
-		}
+		_, _ = subprocess.RunCommand("blkdiscard", "-f", diskPath)
```

Comment:
Should this at least be logged just in case? Debugging issues without any trace on what went wrong could be pretty difficult.

Reply:
True that could be useful.

Reply:
We could make it a debug level event but not anything more visible than that as this code path is hit quite often.